service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -218,16 +226,33 @@ private void configureCollectEvidence() {
         .multicast(new GroupedExchangeAggregationStrategy())
         .process(
             FunctionProcessor.fromFunction(masCollectionService::collectAnnotations)) // call MAS
-        .to(lighthouseEndpoint) // call Lighthouse
-        .end() // end multicast
+        .to(ENDPOINT_LIGHTHOUSE_EVIDENCE)
+        .end()
```

Comment:
Why remove the useful comments?
```suggestion
        .to(ENDPOINT_LIGHTHOUSE_EVIDENCE) // call Lighthouse
        .end() // end multicast
```